debian/ivozprovider-profile-portal.postinst

```diff
@@ -51,7 +51,7 @@ function setup_apache_config()
 
 function setup_gearman_job_server()
 {
-    sed -i -r 's/"(--listen=).*"/"\10.0.0.0"/' /etc/default/gearman-job-server
+    sed -i -r 's/"(--listen=).*"/"\10.0.0.0 --log-file=stderr"/'
```

Comment:
You forgot to add the file at the end of the line, it should be:
```
sed -i -r 's/"(--listen=).*"/"\10.0.0.0 --log-file=stderr"/' /etc/default/gearman-job-server
```

Thanks

Reply:
Fixed